src/transaction/types.h

```diff
@@ -3,9 +3,14 @@
 #include <stddef.h>  // size_t
 #include <stdint.h>  // uint*_t
 
-#define MAX_TX_LEN   510
-#define ADDRESS_LEN  20
-#define MAX_MEMO_LEN 465  // 510 - ADDRESS_LEN - 2*SIZE(U64) - SIZE(MAX_VARINT)
+#include "opcodes.h"
+
+#include "../common/bip32.h"
+
+#define ADDRESS_LEN  25
+#define B58_ADDRESS_LEN  34
+#define TOKEN_UID_LEN 32
+#define TX_INPUT_LEN 35
```

Comment:
Should we `#define PUBKEY_HASH_LEN 20` and use it instead of several `20` along the code?